### DISCUSSION TITLE
How to define new ethnic population?

### DISCUSSION THREAD

**Discussion by misol** [Open-Systems-Pharmacology/Forum#1504] [Questions & Problems] [answered]
Hello there,

Since I was interested in the South Korean population, I tried to make a population on PK-Sim using their **age, weight, and height** distribution data which could be obtained from the national statistics in the country ["A Survey on the Dimension of Korean Persons" in Korean "한국인인체치수조사" by the Ministry of Trade, Industry and Energy (Korean Government)].

Since the Koreans have different weight and height distribution over ages from European, East Asian, and Japanese in [the population options of PK-Sim](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#human), I would like to make a user-defined population (Korean population).

However, I could not find the method to make a user-defined population. Even if I input the median height and weight for an individual (https://github.com/Open-Systems-Pharmacology/Forum/discussions/965), the virtual population referencing the user-defined (height and weight) individual seems to generate one which has characteristics of [the reference population in the PK-Sim (e.g., European, East Asian, and Japanese)](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#human).

Here is a brief comparison of the PK-Sim incorporated populations and South Korean statistics (mean ± standard deviation) for an example. The statistics (mean and SD) for the South Korean population were averaged at 30-34 yr and 35-39 yr groups.

| Weight <br>(kg) | European <br>(PK-Sim generated) | East Asian <br>(PK-Sim generated) | Japanese <br>(PK-Sim generated) | South Korean <br>(Statistics) |
|:---------------:|:-------------:|:-------------:|:-------------:|:-------------:|
| Male <br>30-39 yr   | 75.7 ± 10.3 | 61.2 ± 7.6 | 62.8 ± 6.9 | 78.3 ± 11.5 |
| Female <br>30-39 yr | 63.4 ± 11.1 | 52.6 ± 8.2 | 53.0 ± 7.5 | 58.4 ± 9.5 |

Can I define a user-defined population with new weight-age, height-age, and BMI-age correlation or functions on PK-Sim?

Attached below were for the detailed information.

## Korean statistics
* Body weights grouped by cities: [Korean Body Weights.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/11214903/Korean.Body.Weights.xlsx); Originally downloaded from https://kosis.kr/statHtml/statHtml.do?orgId=350&tblId=DT_35007_N132
* Body weights grouped by sex and age: [Korean Body Weight 3D measured.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/11214922/Korean.Body.Weight.3D.measured.xlsx); Originally downloaded from https://kosis.kr/statHtml/statHtml.do?orgId=115&tblId=DT_115019_503&conn_path=I2
* Height grouped by sex and age: [Korean_Height 3D measured.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/11214934/Korean_Height.3D.measured.xlsx); Originally downloaded from https://kosis.kr/statHtml/statHtml.do?orgId=115&tblId=DT_115019_432&conn_path=I2
* Korean child's growth function: [Korean_child_BW-Height.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/11215334/Korean_child_BW-Height.xlsx); Originally downloaded from https://knhanes.kdca.go.kr/knhanes/sub08/sub08_02.do
* Dev. Korean population for Simcyp (EM/PM...): https://pubmed.ncbi.nlm.nih.gov/30921829/

**Comment by StephanSchaller** [comment]
Dear @misol , thanks for this initiative. This would have to be solved on the database level.
Here is a few forum discussions on the topic:
https://github.com/Open-Systems-Pharmacology/Forum/discussions/875
https://github.com/Open-Systems-Pharmacology/Forum/issues/87

Possibly @Yuri05 and @JanSchlender can support?

**Reply by misol** [reply]
@StephanSchaller if there is a way for it on the GUI PK-Sim, it would be the best for me 🫠

**Reply by StephanSchaller** [reply]
Well, sort of. You can always overwrite the automatically filled default values that appear, when you specify the age.
1. You could create two populations (from europeans?), one male- and one female-only (to have more control over gender-specific specs). 
2. Specify Age range and weight/height-range for each. 
3. Export them (to excel)
4. Merge them in excel
5. re-import the merged population to PK-Sim
 
Best,
Stephan

**Reply by misol** [reply]
>     1. You could create two populations (from europeans?), one male- and one female-only (to have more control over gender-specific specs).
> 
>     2. Specify Age range and weight/height-range for each.
> 
>     3. Export them (to excel)
> 
>     4. Merge them in excel
> 
>     5. re-import the merged population to PK-Sim

Thank you for the reply, but it seems that merging two different virtual populations using Excel and plug-in the merged population into the PK-Sim. It may not make a Korean population, which has different weight, height and BMI from the other bundled populations.

I may be able to make a list of age, weight, and height using a random function of Excel, but it cannot make the other factors, including Ontogeny factor (albumin, alpha1-acid glycoprotein), organ volumes (fixed relative volume?, I don't know), blood flow, and so on.

**Reply by msevestre** [reply]
No need to merge in excel. PKSim allows you to merge population right from the UI.

**Comment by StephanSchaller** [comment, marked as answer]
> I may be able to make a list of age, weight, and height using a random function of Excel, but it cannot make the other factors, including Ontogeny factor (albumin, alpha1-acid glycoprotein), organ volumes (fixed relative volume?, I don't know), blood flow, and so on.

Indeed, this is what I was referring to when I said, that this would require setting up a database for a Korean population similar to what now exists for Japanese and European, etc. populations in PK-Sim.

If you have all the data available, this should not be too much effort.

**Reply by misol** [reply]
> > I may be able to make a list of age, weight, and height using a random function of Excel, but it cannot make the other factors, including Ontogeny factor (albumin, alpha1-acid glycoprotein), organ volumes (fixed relative volume?, I don't know), blood flow, and so on.
> 
> Indeed, this is what I was referring to when I said, that this would require setting up a database for a Korean population similar to what now exists for Japanese and European, etc. populations in PK-Sim.
> 
> If you have all the data available, this should not be too much effort.

Can I understand this as: there are no ways to make a 'user-defined' population on PK-Sim which has no similarity to the pre-loaded populations?

Then, I may not be able to make a virtual Korean population on PK-Sim if I don't think the distribution of the observed Korean population is similar to the other pre-loaded populations.

That's okay. Then, if I want to incorporate every aspect that I can get about the Korean population in PK-Sim, it would be necessary to develop some R or Python codes to generate an Excel file for a virtual Korean population. I will try to do that. I will refer to the algorithms on [the PK-Sim manual](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-populations)

**Comment by JanSchlender** [comment]
Hello Misol,

As far as I can see, you only have body measures of the Korean population. These can be defined when generating a population in the PK-SIm Gui by defining a range matching your desired Korean population measures. Please do not do this in Excel, since the underlying anatomical and physiological information will only be scaled when a population is generated in PK-SIm. In case you change e.g. body weight in Excel afterwards, the sum of all organs would not match the new body weight.

You can also change ontogenies in the GUI within the individual under the respective ADME protein.

Things get a bit trickier once you want to change organ sizes, blood flows,... This can only be altered on a database level. Again, keep in mind the interdependencies of organ volumes (sum needs to add up to the body weight) and blood flows (sum needs to match Cardiac output). Details can be studied in the following references:
https://link.springer.com/article/10.1007/s40262-016-0422-3 
https://link.springer.com/article/10.1007/s40262-017-0539-z
https://link.springer.com/article/10.2165/00003088-200645100-00005

The easiest way to get this in PK-Sim is writing a script which modifies the exported population csv and re-import the file again. When conceptualising the script, I suggest to work with factors as deltas from the pre-selected population information. This way you are still making use of the population generation algorithm in PK-Sim.
https://link.springer.com/article/10.1007/s10928-007-9053-5

**Reply by misol** [reply]
Hello @JanSchlender 

Thank you for the detailed description of the PK-Sim-generated virtual populations.

I understand that every parameter should have integrity, such as mass-balanced.
I already left a reply that I will write a script file (e.g., R- or Python-based) to generate virtual populations.

Because this topic started from my personal interest (because I am a South Korean person), I planned to follow this topic on a long time scale. Below is a summary of my thinking.

1. Sums of tissue weight and perfusion rate are linked to the total body weight and cardiac outputs. There would be other related parameters. I need to:
    - find Korean-specific _relative organ weight_ (e.g., X-ray image; https://pubmed.ncbi.nlm.nih.gov/16332919/ ; https://koreascience.kr/article/JAKO201215734995909.pdf)
    - find Korean-specific tissue blood flow (or _relative blood flows_) and/or cardiac output.
    - find age- and sex- related parameters
    - find Korean-specific distributions for the expression of drug-metabolizing enzymes (e.g., https://pubmed.ncbi.nlm.nih.gov/30921829/) and transporters.
    - If I cannot find a difference or cannot find practical meaning, or if I can think the "_relative_" value is not different among ethnic groups, I will assume the same value among ethnic groups (However, the absolute value will be different)
2. Model validation
    - find clinical trial data which shows different concentration-time profiles between Korean and the other ethnic group
    - Make PBPK models for the case drugs or compounds (above) and see if the user-defined virtual population regenerates the difference.

I will not use only _Excel_. If there is a clear algorithm, I think I can make a code to execute it. Since there are published algorithms for generating virtual populations, I can adapt them for the virtual Korean populations.

Thank you again for the detailed reply 👍

**Reply by JanSchlender** [reply]
Hello Misol,
This is a good outline. Fulfilling all steps seems really ambitious and may take month and so you might be fine with going for the adult population first. Then, you can already run ethnic bridging simulations. Feel free to reach out anytime for technical details.

When building the Japanese population, I came across Korean data as well. Some sources which may help as a starting point for you:

- Perfusion/Cardiac Output
https://onlinelibrary.wiley.com/doi/full/10.1002/clc.20477
https://link.springer.com/article/10.1007/s10554-012-0150-1
https://journals.lww.com/annalsplasticsurgery/abstract/1997/12000/mapping_of_the_human_body_skin_with_laser_doppler.7.aspx

- Organs
https://synapse.koreamed.org/articles/1019150
https://kmbase.medric.or.kr/KMID/0363720090420040219
https://academic.oup.com/rpd/article-abstract/118/3/275/1602593
https://onlinelibrary.wiley.com/doi/abs/10.1002/ajpa.1330220103 (rather old study)
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3220241/

- Growth charts
https://koreascience.kr/article/JAKO200810102403465.pdf
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5976563/
https://synapse.koreamed.org/articles/1082652

- KNHANES data and publications e.g.
https://www.sciencedirect.com/science/article/pii/S2214762414000139
https://search.informit.org/doi/abs/10.3316/INFORMIT.104794386370993
https://academic.oup.com/jcem/article/97/9/3310/2537072
https://synapse.koreamed.org/articles/1022209

**Reply by misol** [reply]
Hello @JanSchlender 

Thank you for the articles, and I will share the progress if there is good news from it.
As you said, it's going to take some time (months) 😄

**Reply by BethTchai** [reply]
Hi, did you face any other issues that might be useful for others to know? I am trying to do the same for an Indian or South Asian population. I just wanted to know whether you faced other relevant issues in doing this.

**Comment by misol** [comment]
Just in case, I am no longer proceeding with this topic due to my personal reasons.
Please feel free to make advances in this customized population topic.
I am leaving this message because I have received a direct message related to this thread on the other platform.
